### PR TITLE
Checkout: Add a tooltip to the CVV field

### PR DESCRIPTION
**Pull request by codebykat**
This PR will address an issue raised in https://github.com/Automattic/wp-calypso/issues/22659, by adding an explanatory tooltip to the CVV field.

### CODE REVIEW

File: client/components/credit-card-form-fields/index.jsx [hunk 1]
```diff
@@ -212,7 +212,7 @@ export class CreditCardFormFields extends React.Component {
 				<div className={ creditCardFormFieldsExtrasClassNames }>
 					{ this.createField( 'expiration-date', Input, {
 						inputMode: 'numeric',
-						label: translate( 'MM/YY', {
+						label: translate( 'Expiry: MM/YY', {
```
Comment (a8ci18n) [inline client/components/credit-card-form-fields/index.jsx]: :information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?

Comment (yoavf) [reply client/components/credit-card-form-fields/index.jsx]: This make sense - we can lose the context now that the string itself is more explicit

**Comment by gwwar** [comment]
@codebykat was this one ready to 👀 ?

**Comment by codebykat** [comment]
Moved the expiry date half of this PR into https://github.com/Automattic/wp-calypso/pull/24181 to be merged separately.

I ran out of time to sort out the tooltip here. In order to position it correctly vis-a-vis the form input and label, we will need to create a new form input component (similar to the `CreditCardNumberInput` component). This will allow us to wrap a container around the label and popup and we'd then reposition the tooltip relatively with CSS.

**Pull request closed without merging**